[["/usr/lib/python3/dist-packages/service_identity/__init__.py",[false,[["CertificateError",true,2,["CertificateError","/usr/lib/python3/dist-packages/service_identity/exceptions.py",5]],["SubjectAltNameWarning",true,2,["SubjectAltNameWarning","/usr/lib/python3/dist-packages/service_identity/exceptions.py",5]],["VerificationError",true,2,["VerificationError","/usr/lib/python3/dist-packages/service_identity/exceptions.py",5]],["__title__",true,13,null],["__author__",true,13,null],["__license__",true,13,null],["__copyright__",true,13,null],["__all__",true,13,null],["__getattr__",true,12,null]]]]]